### ISSUE TITLE
Persistent storage of redis

### ISSUE THREAD

**Issue by joepio** [ontola/raadstalk#14]
Currently, redis seems to lose all data when the containers stop. We need some form of persistence.

**Comment by jurrian** [comment]
In the docker-compose file it seems that redis does have a volume for data.

**Comment by joepio** [comment]
Yeah, but still it was very unreliable. I hope the previous fix did the trick.

**Comment by joepio** [comment]
Running `./renew` (which restarts compose) seems to erase the data as well. Issue is not yet fixed.

**Comment by jurrian** [comment]
I remember it had something to do with overriding the redis settings, is that still the case?

**Comment by jurrian** [comment]
> I remember it had something to do with overriding the redis settings, is that still the case?

Nevermind, that was already addressed in bbda7d8a6b089beac1453be80539f6bd59e0b712
